src/ui/markup/gptk-main-template.html

```diff
@@ -91,6 +91,7 @@
       <button type="button" id="unFavorite" title="Removed From Favorites">Un-Favorite</button>
       <button type="button" id="lock" title="Move to Locked Folder">Move to Locked Folder</button>
       <button type="button" id="unLock" title="Move out of Locked Folder">Move out of Locked Folder</button>
+      <button type="button" id="copyDescFromOther" title="Copy Description from 'Other' Field">Copy Description from 'Other'</button>
     </div>
 
     <div class="to-existing-container">
```

Comment:
```suggestion
      <button type="button" id="unFavorite" title="Removed From Favorites">Un-Favorite</button>
      <button type="button" id="lock" title="Move to Locked Folder">Move to Locked Folder</button>
      <button type="button" id="unLock" title="Move out of Locked Folder">Move out of Locked Folder</button>
      <button
        type="button"
        id="copyDescFromOther"
        title="Copies captions embedded in the photos themselves (i.e. EXIF data) to the photo description.\nSuch captions otherwise only show up in the Info tab as 'Other'"
      >
        Copy Description from 'Other'
      </button>
    </div>

    <div class="to-existing-container">
```

add explanation to button's title

Reply:
I took your suggestion but updated the comment to say that copying occurs only if the current description is empty.  (I had kept the title short before to match the style of the existing buttons, but this seems a lot more helpful.)